### PR TITLE
Issue #72: Created aCircuitBreakerMetrics class which can be used to …

### PR DESCRIPTION
**Pull request by RobWin**
…export CircuitBreaker.Metrics as Dropwizard Metrics Gauges.

### CODE REVIEW

File: resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/CircuitBreakerMetrics.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            metricRegistry.register(name("resilience4j.circuitbreaker", circuitBreaker.getName(), "buffered_max"),
+                    (Gauge<Integer>) () -> circuitBreaker.getMetrics().getMaxNumberOfBufferedCalls());
+            }
+        );
+    }
+
+    public static CircuitBreakerMetrics of(CircuitBreakerRegistry circuitBreakerRegistry) {
+        return new CircuitBreakerMetrics(circuitBreakerRegistry.getAllCircuitBreakers());
+    }
+
```
Comment (storozhukBM) [inline resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/CircuitBreakerMetrics.java]: I'd add factory method for collection of circuit breakers

File: resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/CircuitBreakerMetrics.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+/**
+ * An adapter which exports {@link CircuitBreaker.Metrics} as Dropwizard Metrics Gauges.
+ */
+public class CircuitBreakerMetrics implements MetricSet{
+
+    private final MetricRegistry metricRegistry = new MetricRegistry();
+
+    private CircuitBreakerMetrics(CircuitBreakerRegistry circuitBreakerRegistry){
+        circuitBreakerRegistry.getAllCircuitBreakers().forEach(circuitBreaker -> {
+            metricRegistry.register(name(CircuitBreaker.class, circuitBreaker.getName(), "successful"),
```
Comment (storozhukBM) [inline resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/CircuitBreakerMetrics.java]: move `circuitBreaker.getMetrics()` and `circuitBreaker.getName()` to separate variables on the start of the lambda

File: resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/CircuitBreakerMetrics.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+/**
+ * An adapter which exports {@link CircuitBreaker.Metrics} as Dropwizard Metrics Gauges.
+ */
+public class CircuitBreakerMetrics implements MetricSet{
+
+    private final MetricRegistry metricRegistry = new MetricRegistry();
+
+    private CircuitBreakerMetrics(Seq<CircuitBreaker> circuitBreakers){
+        circuitBreakers.forEach(circuitBreaker -> {
+            metricRegistry.register(name("resilience4j.circuitbreaker", circuitBreaker.getName(), "successful"),
```
Comment (storozhukBM) [inline resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/CircuitBreakerMetrics.java]: You can move `circuitBreaker.getMetrics()` and `circuitBreaker.getName()` to separate variables on the start of the lambda.

**Comment by RobWin** [comment]
Thx. Done.

**Pull request merged**